File: src/NBdbm/Classes/Interfaces/iManager.vb
Option Explicit On
Namespace Interfaces

    Public Interface iManager

        Function createDB(ByVal commandString As String) As Object

        Function createTable(ByVal commandString As String) As Object

        Function createView(ByVal commandString As String) As Object

        Function createField(ByVal commandString As String) As Object

        Function readDB(ByVal SQL As String) As Data.DataView

    End Interface

End Namespace

File: src/NBdbm/CS/Classes/Interfaces/iManager.cs
using System.Data;
using System.Collections;
using Microsoft.VisualBasic;
using System.Diagnostics;
using System;

namespace NBdbm
{
	namespace Interfaces
	{
		
		public interface iManager
		{
			
			object createDB(string commandString);
			
			object createTable(string commandString);
			
			object createView(string commandString);
			
			object createField(string commandString);
			
			System.Data.DataView readDB(string SQL);
			
		}
		
	}
}
